### ISSUE TITLE
JDK20 requires implementation of JVM_VirtualThreadHideFrames()

### ISSUE THREAD

**Issue by keithc-ca** [eclipse-openj9/openj9#16012]
Uses introduced in [8293613: need to properly handle and hide tmp VTMS transitions](https://github.com/ibmruntimes/openj9-openjdk-jdk/commit/053a7d3c2fceda84796237fac0d610458cad834c).

Staging builds are currently failing (e.g. [s390x](https://openj9-jenkins.osuosl.org/job/Build_JDKnext_s390x_linux_OpenJDK/351/console)):
```
04:49:16  === Output from failing command(s) repeated here ===
04:49:16  * For target support_native_java.base_libjava_BUILD_LIBJAVA_link:
04:49:16  /home/jenkins/workspace/Build_JDKnext_s390x_linux_OpenJDK/build/linux-s390x-server-release/support/native/java.base/libjava/VirtualThread.o:(.data.rel.methods+0x70): undefined reference to `JVM_VirtualThreadHideFrames'
04:49:16  collect2: error: ld returned 1 exit status
```

**Comment by pshipton** [comment]
@tajila @fengxue-IS @babsingh fyi
For now Keith will implement a stub so we can continue building jdk20.

**Comment by fengxue-IS** [comment]
@thallium is taking a look at this issue

**Comment by tajila** [comment]
@thallium @fengxue-IS how close is this to completion?

**Comment by thallium** [comment]
@tajila We are mainly waiting for #16654 to be merged, #17125 is just some refactor.
